### DISCUSSION TITLE
TSInstallSync for only the ensure_installed parsers?

### DISCUSSION THREAD

**Discussion by TravisDart** [nvim-treesitter/nvim-treesitter#6732] [Q&A]
I've created a Docker container that loads nvim (nvChad). Currently, I install all nvim requirements with this command:
```
nvim --headless "+Lazy! sync" "+Lazy! load nvim-treesitter" "+TSInstallSync vim lua vimdoc html css python" +qa!
```

I'm wanting to add more Treesitter plugins, so this list is going to get pretty long. Plus, I have to keep it in sync with my nvim config. I don't really want to do `TSInstallSync all`, either.

Is there a way to just install the Treesitter plugins listed in the `ensure_installed` list in the config?

**Comment by clason** [comment]
No, sorry. You can check the code to find the underlying Lua function and call that with the same table you pass to ensure_installed.
